app/views/layouts/application.html.erb

```diff
@@ -18,7 +18,10 @@
   <%= render 'layouts/alerts' %>
 </header>
 <main>
-  <%= yield %>
+  </br>
```

Comment:
Better to use an extra margin for the container than to use `</br>`

Reply:
It also occurs elsewhere.

Reply:
< /br> are default used in Devise forms to separate form fields. IDK if we really need to replace them.

Reply:
I think that you should use css rather than tags for such things.

Reply:
I will change by </br>, don't worry. :D
Just asking if we should do something about the Devise ones.